src/shadow-embed.js

```diff
@@ -548,6 +548,31 @@ export class ShadowDomWriterStreamer {
     return this.success_;
   }
 
+  /** @override */
+  get closed() {
+    return this.eof_;
+  }
+
+  /** @override */
+  get desiredSize() {
+    return 1; // noop
```

Comment:
Yeah, `eof_` should definitely yield 0. Feels like the API is a bit duplicative, but it is what it is. But the meaning of size still evades me here. Is that byte size? Char size? Queue size? Unicorn size?